#!/usr/bin/env python

PACKAGE = "rossim"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('minimo',double_t,0,'valor 100 en el mapa',0.3,0,1)
gen.add('maximo',double_t,0,'valor 0 en el mapa',0.5,0,1)

gen.add('spacing_cm',int_t,0,'Espaciado de malla',45,0,None)
gen.add('search_radius_cells',int_t,0,'Tamano de malla',30,0,None)
gen.add('turn_weight',int_t,0,'Penalizacion giro en trayectoria',0,0,None)

exit(gen.generate(PACKAGE, "rossim", "_rossim_"))